packages/asset-buyer/docs/reference.mdx

```diff
@@ -0,0 +1,1352 @@
+> # Class: AssetBuyer
+
+## Hierarchy
+
+* **AssetBuyer**
+
+## Index
+
+### Constructors
+
+* [constructor](#constructor)
+
+### Properties
+
+* [expiryBufferSeconds](#expirybufferseconds)
+* [networkId](#networkid)
+* [orderProvider](#orderprovider)
+* [orderRefreshIntervalMs](#orderrefreshintervalms)
+* [provider](#provider)
+
+### Methods
+
+* [executeBuyQuoteAsync](#executebuyquoteasync)
+* [getAvailableAssetDatasAsync](#getavailableassetdatasasync)
+* [getBuyQuoteAsync](#getbuyquoteasync)
+* [getBuyQuoteForERC20TokenAddressAsync](#getbuyquoteforerc20tokenaddressasync)
+* [getLiquidityForAssetDataAsync](#getliquidityforassetdataasync)
+* [getOrdersAndFillableAmountsAsync](#getordersandfillableamountsasync)
+* [getAssetBuyerForProvidedOrders](#static-getassetbuyerforprovidedorders)
+* [getAssetBuyerForStandardRelayerAPIUrl](#static-getassetbuyerforstandardrelayerapiurl)
+
+## Constructors
+
+###  constructor
+
+\+ **new AssetBuyer**(`supportedProvider`: `SupportedProvider`, `orderProvider`: [OrderProvider](#class-assetbuyer)*
```

Comment:
This seems to be missing a closing parenthesis and looks like its picked up the incorrect type for the returns. 
`[OrderProvider](#class-assetbuyer)`

Renders:
> new AssetBuyer(supportedProvider: SupportedProvider, orderProvider: OrderProvider*

Reply:
Fixed this issue. Thanks for spotting!